src/commands.cpp

```diff
@@ -162,6 +165,89 @@ CON_COMMAND_CHAT(stopsound, "stop weapon sounds")
 	ClientPrint(player, HUD_PRINTTALK, " \7[CS2Fixes]\1 You have toggled weapon effects.");
 }
 
+CON_COMMAND_CHAT(ztele, "teleport to spawn")
+{
+	if (!player)
+		return;
+
+	//Count spawnpoints (info_player_counterterrorist & info_player_terrorist)
+	SpawnPoint* spawn = nullptr;
+	CUtlVector<SpawnPoint*> spawns;
+	while (nullptr != (spawn = (SpawnPoint*)UTIL_FindEntityByClassname(spawn, "info_player_")))
+	{
+		if (spawn->m_bEnabled())
+		{
+			spawns.AddToTail(spawn);
+		}
+	}
+
+	//Pick and get position of random spawnpoint
+	int randomindex = rand() % spawns.Count()+1;
```

Comment:
I've read that `srand()` needs to be called once per application run for proper randomness. This is already done in [@random targeting](https://github.com/Source2ZE/CS2Fixes/blob/ad0891501918bbd10c0c62f9af42cb657977a629/src/playermanager.cpp#L156-L160), but should probably be moved to plugin startup now that `rand()` is being used in several locations.